homeassistant/components/utility_meter/sensor.py

```diff
@@ -241,7 +242,16 @@ async def async_added_to_hass(self):
         """Handle entity which will be added."""
         await super().async_added_to_hass()
 
-        if self._period == HOURLY:
+        if self._period == QUARTER_HOURLY:
+            for quarter in [0, 1, 2, 3]:
```

Comment:
Use range function instead of a list of integers.
```suggestion
            for quarter in range(4):
```